frontend/src/store/store.js

```diff
@@ -78,32 +78,38 @@ export const actions = {
   },
 }
 
-export const store = new Vuex.Store({
-  plugins: [
-    createPersistedState({
-      storage: window.localStorage,
-    }),
-  ],
-  state: {
-    email: '',
-    language: null,
-    firstName: '',
-    lastName: '',
-    username: '',
-    description: '',
-    token: null,
-    isAdmin: false,
-    coinanimation: true,
-    newsletterState: null,
-    community: {
-      name: '',
+let store
+
+try {
+  store = new Vuex.Store({
+    plugins: [
+      createPersistedState({
+        storage: window.localStorage,
+      }),
+    ],
+    state: {
+      email: '',
+      language: null,
+      firstName: '',
+      lastName: '',
+      username: '',
       description: '',
+      token: null,
+      isAdmin: false,
+      coinanimation: true,
+      newsletterState: null,
+      community: {
+        name: '',
+        description: '',
+      },
+      hasElopage: false,
+      publisherId: null,
     },
-    hasElopage: false,
-    publisherId: null,
-  },
-  getters: {},
-  // Syncronous mutation of the state
-  mutations,
-  actions,
-})
+    getters: {},
+    // Syncronous mutation of the state
+    mutations,
+    actions,
+  })
+} catch (error) {}
```

Comment:
lets not do this - its better to have the error on the console in case things go wrong?! e.g. esoteric browsers like edge behave completely irrational. Or does then to redirect not trigger because the js dies? if so please log the error to the console here in order to allow debugging